pepper-apis/studybuilder-cli/studies/osteo/activities/prequal.conf

```diff
@@ -34,10 +34,8 @@
                 {
                   "name": "prompt_PREQUAL_SELF_DESCRIBE",
                   "translations": [
-                    {
-                      "language": "en",
-                      "text": "First, who is signing up for the Osteosarcoma Project? <br><em>Check all that apply</em>"
-                    }
+                    { "language": "en", "text": ${i18n.en.prequal.describe_prompt} },
+                    { "language": "es", "text": ${i18n.es.prequal.describe_prompt} }
```

Comment:
Shouldn't there be an i18n/es.conf in this PR as well?

Reply:
es.conf exists and once we get new es translations from CMI, we will updates es.conf

Reply:
No changes to es.conf until we get translations from cmi

Reply:
I see!!